src/components/DeleteMovie/DeleteMovie.tsx

```diff
@@ -0,0 +1,24 @@
+import { useState } from "react";
+import Modal from "../Modal/Modal";
+import classes from "./DeleteMovie.module.css";
+
+const DeleteMovie: React.FC = () => {
+  const [modal, setModal] = useState(false);
+
+  const toggle = () => setModal(!modal);
+
+  return (
+    <>
+      <button className={classes.btn} onClick={() => toggle()}>
+        Delete
+      </button>
+      <Modal show={modal} close={toggle} title="Delete Movie">
+        <form>
+          <p>Are you sure you want to delete this movie?</p>
+        </form>
+      </Modal>
```

Comment:
Why did you add this form wrapper here?

Reply:
Just a habit. Since the user will take action and submit for deletion I used a form. But thanks for the warning, right know submission and content in a different place. So I have to think about it.

Reply:
Extra Dom elements is a bad practice, because it affects performance & optimization. Please, remove unnecessary tag.

Reply:
Removed ✅